### PR TITLE
Displays big image

### PR DESCRIPTION
**Pull request by maxiaoping**
View Original Image

### CODE REVIEW

File: app/src/main/java/com/seafile/seadroid2/data/SeafPhoto.java [hunk 1]
```diff
@@ -8,6 +8,8 @@
 public class SeafPhoto implements Serializable {
     public static final long serialVersionUID = 0L;
 
+    /** download shows tatus */
+    private boolean showStatus;
```
Comment (freeplant) [inline app/src/main/java/com/seafile/seadroid2/data/SeafPhoto.java]: showStatus -> downloaded

File: app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java [hunk 2]
```diff
@@ -415,5 +447,38 @@ private void removePageAndRefreshView() {
...
+            public void onFileDownloadProgress(int taskID) {
+
+            }
+
+            @Override
+            public void onFileDownloaded(int taskID) {
+                if (mGalleryAdapter != null) {
+                    mGalleryAdapter.downloadPhoto();
+                }
+                if(progressDialog!=null){
```
Comment (freeplant) [inline app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java]: `if(progressDialog!=null){`  -> `if (progressDialog  != null) {`

File: app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java [hunk 3]
```diff
@@ -415,5 +447,38 @@ private void removePageAndRefreshView() {
...
+                    progressDialog.dismiss();
+                }
+
```
Comment (freeplant) [inline app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java]: 多空行

File: app/src/main/java/com/seafile/seadroid2/data/SeafPhoto.java (hunk 1 above)
Comment (maxiaoping) [reply app/src/main/java/com/seafile/seadroid2/data/SeafPhoto.java]: ok

File: app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java [hunk 4]
```diff
@@ -415,5 +447,38 @@ private void removePageAndRefreshView() {
...
+            }
+
+            @Override
+            public void onFileDownloadFailed(int taskID) {
+                count++;
+                if (count < TALLY) {
+                    downloadFile(repoID, dirPath, fileName);
+                }else {
```
Comment (freeplant) [inline app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java]: 少空格

File: app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java [hunk 5]
```diff
@@ -415,5 +447,38 @@ private void removePageAndRefreshView() {
...
+                    if(progressDialog!=null){
```
Comment (freeplant) [inline app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java]: 同上

File: app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java [hunk 6]
```diff
@@ -322,6 +343,17 @@ public void hideOrShowToolBar() {
 
     }
 
+    public void hideOrShowToolBarDownload() {
+        mDownloadBtn.setVisibility(View.GONE);
+    }
+
+    private void downloadFile(String repoID, String dirPath, String fileName) {
+        progressDialog = ProgressDialog.show(this,"", getString(R.string.notification_download_started_title));
```
Comment (freeplant) [inline app/src/main/java/com/seafile/seadroid2/ui/activity/GalleryActivity.java]: 少空格

**Pull request merged**